packages/@react-aria/dnd/src/DragManager.ts

```diff
@@ -69,8 +70,10 @@ export function beginDragging(target: DragTarget, formatMessage: (key: string) =
   dragSession = new DragSession(target, formatMessage);
   requestAnimationFrame(() => {
     dragSession.setup();
-
-    if (getDragModality() === 'keyboard') {
+    if (
+      getDragModality() === 'keyboard' ||
+      (getDragModality() === 'touch' && getInteractionModality() === 'virtual')
```

Comment:
Out of curiosity, is the `&& getInteractionModality() === 'virtual'` necessary (technically beginDragging only fires on drag events from keyboard/virtual drags due to useDrag's onPress)?

Reply:
On second thought, checking the interactionModality as well feels safer